### PR TITLE
[T4A6][T14-B1] Tan Min Rong Samson

### PR DESCRIPTION
**Pull request by samsontmr**
Ready for review.

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    @Test
+    public final void test_isAnyNull_singleObject() {
+        Object object = new Object();
+        assertFalse(Utils.isAnyNull(object));
+    }
+    
+    @Test
+    public final void test_elementsAreUnique_singleObject() {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:36]: the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:3]: don't import everything. import only what is necessary.

**Comment by okkhoy** [comment]
@samsontmr 
Some comments added. Please ack & close the PR after reading comments.

**Comment by samsontmr** [comment]
Acknowledged

**Pull request closed without merging**